deposit_contract/contracts/validator_registration.v.py

```diff
@@ -56,14 +60,14 @@ def get_deposit_root() -> bytes32:
 @payable
 @public
 def deposit(deposit_input: bytes[512]):
-    assert msg.value >= as_wei_value(MIN_DEPOSIT_AMOUNT, "gwei")
-    assert msg.value <= as_wei_value(MAX_DEPOSIT_AMOUNT, "gwei")
+    deposit_amount: uint256 = msg.value / as_wei_value(1, "gwei")
+    
+    assert msg.value >= deposit_amount
```

Comment:
should be:
```suggestion
    assert deposit_amount >= MIN_DEPOSIT_AMOUNT
```

Reply:
sorry I missed this ! made the changes, all the tests pass now